docs/user-guide/src/contributing/services_contacts.md

```diff
@@ -0,0 +1,118 @@
+
+### IDS Wazuh
+
+[Wazuh](https://wazuh.com/) is a "security platform" used to keep the bare metal lab secure.
+See [Nordix wiki](https://wiki.nordix.org/pages/viewpage.action?pageId=47546865).
+
+### Jenkins
+
+The [Nordix Jenkins instance](https://jenkins.nordix.org/) is used for all "heavier" tests and build jobs (e.g. integration tests, e2e tests and node image building jobs).
+It is managed by the Nordix admins, but we provide the Jenkins agents, pipelines and configure secrets.
+
+The jobs are defined using [Jenkins Job Builder](https://jenkins-job-builder.readthedocs.io/en/latest/) and are stored in the [cicd repo on the Nordix Gerrit](https://gerrit.nordix.org/admin/repos/infra/cicd,general).
+Each job references a pipeline in [project-infra](https://github.com/metal3-io/project-infra/tree/main/jenkins/jobs) or [metal3-dev-tools](https://github.com/Nordix/metal3-dev-tools/tree/master/ci/jobs) that is executed by Jenkins when the job is triggered.
+
+### Jump hosts
+
+We have jump hosts in Cleura which are used to avoid having to expose every developer VM to the internet.
+See [this wiki page](https://wiki.nordix.org/display/IN/Jumphost+Setup).
+
+### Bare metal lab
+
+We have access to a server rack with actual physical servers.
+This is used for some CI jobs as well as for manual testing.
+We are responsible for the software on these machines but not for the hardware itself.
+More details can be found on the [BML wiki](https://wiki.nordix.org/display/CPI/Bare+Metal+Lab).
+
+## Contact information to those we rely on
+
+When things break, it is often because of issues in the underlying infrastructure.
+In these cases we need to know who to contact to report the issues and where to look for known issues (e.g. status pages).
+
+### Cleura
+
+Contanct: <support@cleura.com>
+
+Status page: <https://www.cnstatus.com/>
+
+Cleura (former CityCloud) is the cloud platform we run most of our things on.
+All the Jenkins jobs, Prow and all developer VMs are here.
+They basically donate the resources to us and the normal support email is available if we need to reach them.
+
+### Nordix services
+
+Contact: <discuss@lists.nordix.org>
+
+The [Nordix association](https://www.nordix.org/) provides several services that we rely on.
+
+- [Gerrit](https://gerrit.nordix.org/): code repositories.
+- [Wiki](https://wiki.nordix.org/): gather useful information.
+- [Jenkins](https://jenkins.nordix.org/): tests and build jobs.
+- [Artifactory](https://artifactory.nordix.org/): store build artifacts.
+- [Jira](https://jira.nordix.org): ticket system.
+
+### Quay
+
+Contact: <support@quay.io>
+
+Status page: <https://status.quay.io/>
+
+We use [Quay](https://quay.io) to store the container images for all projects.
+
+### Bare metal lab hardware contact
+
+Contact: Ian Kumlien (<ian.kumlien@ericsson.com>)
+
+Note this is for hardware issues.
+We are responsible for the software.
+See the [BML wiki](https://wiki.nordix.org/display/CPI/Bare+Metal+Lab).
```

Comment:
I am not sure if this is needed, because it is needed only for the team that manages the BML but not for everyone